api/api/utils/photon.py

```diff
@@ -11,6 +11,8 @@
 import requests
 import sentry_sdk
 
+from api.utils.tallies import get_monthly_timestamp
+
```

Comment:
Just a small nit/future improvement/consideration (depending on how general we want to think about "tallying", if it's an idea we'll continue to expand and do elsewhere) I wonder if somehow this error handling code could be moved as a separate unit into the tallies module. It would at least make it simpler to read the general Photon request lifecycle without getting muddled down in the error handling.